Classes/game.py

```diff
@@ -316,6 +316,31 @@ def setStartRoom(self):
                 self.player.setLocation(room)
                 return
 
+    def addToItemStorage(self, item):
+        """
+        Adds an Item to game storage.
+        """
+        self.itemStorage.append(item)
+
+    def removeFromItemStorage(self, item):
+        """
+        Removes an Item from game storage.
+        Returns True if successful, False if not.
+        """
+        if item in self.itemStorage:
+            self.itemStorage.append(item)
+            return True
+        return False
+
```

Comment:
It looks like this function adds an item to ItemStorage rather than removing from it. (line 331)